bundles/remote_services/remote_service_admin_shm_v2/README.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Remote Service Admin SHM
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+   
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Remote Service Admin SHM
+
+The Celix Remote Service Admin SHM bundle implement OSGi remote service using shared memory.
+In order to reuse the (de)serialization features of remote service admin,
+the remote service admin will be realized by ‘rsa_shm’ and 'rsa_json_rpc'(see [Remote Service Admin RPC Using JSON](../rsa_rpc_json/README.md)) bundles.
+The ‘rsa_shm’ bundle is used to realize ‘remote_service_admin_service_t’ and IPC.
+The ‘rsa_json_rpc’ bundle is used to realize (de)serialization and Creating service endpoint/proxy.
+
+### Supported Platform
+- Linux
+
+### Properties/Configuration
+
+| **Properties** | **Type** | **Description**|
+|----------------|----------|----------------|
+| **rsaShmPoolSize**  | long     | The RSA SHM pool size in bytes. Its value should be greater than 8192 bytes.(default 256KB)                                                               |
```

Comment:
Here, we suggest that the value of RSA_SHM_POOL_SIZE should be greater than or equal to 8192 bytes, Because current memory pool ctrl block(control_t of [tlsf](https://github.com/apache/celix/tree/master/bundles/remote_services/remote_service_admin_shm_v2/shm_pool/src/tlsf)) size is 6536 bytes.

I've added a corresponding note to the documentation.